src/Features/LanguageServer/Protocol/Handler/CodeActions/CodeActionHelpers.cs

```diff
@@ -143,29 +214,29 @@ internal static class CodeActionHelpers
 
                 return nestedActions.ToArray();
             }
+        }
 
-            static LSP.Diagnostic[]? GetApplicableDiagnostics(LSP.CodeActionContext context, IUnifiedSuggestedAction action)
+        private static LSP.Diagnostic[]? GetApplicableDiagnostics(CodeActionContext context, IUnifiedSuggestedAction action)
```

Comment:
This method is unchanged. Only move it from local static method to class static method.